release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -2780,3 +2780,30 @@ link:https://access.redhat.com/solutions/6934361[{product-title} 4.9.29 containe
 ==== Updating
 
 To upgrade an existing {product-title} 4.9 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+
+[[ocp-4-9-31]]
```

Comment:
You have two brackets here [[ocp-4-9-31]] and it should be [id="ocp-4-9-31"] .

Reply:
Can you fix 4.9.29 while you are at it see line 2760

Reply:
Got it! removed brackets for 4.9.31 and 4.9.29!